tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -520,6 +542,50 @@ def _test_data_repository_task(remote_command_executor, mount_dir, bucket_name,
     assert_that(file_permissions).is_equal_to("0100777")
 
 
+def _test_storage_capacity(remote_command_executor, mount_dir, storage_capacity):
+    logging.info("Test FSx storage capacity")
+    if storage_capacity:
+        result = remote_command_executor.run_remote_command(
+            f"df -BG | grep {mount_dir}" + "| awk '{print$2}' | tr -d -c 0-9"
+        )
+        assert_that((storage_capacity - int(result.stdout)) // storage_capacity).is_less_than(0.2)
+
+
+def _test_imported_file_chunch_size(fsx_fs_id, imported_file_chunk_size, region):
+    logging.info("Test FSx ImportedFileChunkSize setting")
+    if imported_file_chunk_size:
+        assert_that(get_imported_chunch_size(fsx_fs_id, region)).is_equal_to(imported_file_chunk_size)
+
+
+def _test_weekly_maintenance_start_time(fsx_fs_id, weekly_maintenance_start_time, region):
+    logging.info("Test FSx weekly maintenance start time setting")
+    if weekly_maintenance_start_time:
+        assert_that(get_weekly_maintenance_start_time(fsx_fs_id, region)).is_equal_to(weekly_maintenance_start_time)
+
+
+def get_imported_chunch_size(fsx_fs_id, region):
+    logging.info("Getting ImportedFileChunkSize from DescribeFilesystem API.")
+    return (
+        boto3.client("fsx", region_name=region)
```

Comment:
Can we just describe the file system once and parse different info out of the result in different tests?

Please double check for other assertions/test cases that may need to describe the file system as well

Reply:
Done